src/pages/docs/deployments/kubernetes/kustomize/index.md

```diff
@@ -15,33 +15,41 @@ This step allows you to source your kustomize files from git, perform variable s
 
 ## Recommended usages for this step
 
-We list three different scenarios below to help you figure out what is the best setup for you to deploy your applications to a Kubernetes cluster with Octopus.
+We list a few different scenarios below to help you figure out what is the best setup for you to deploy your applications to a Kubernetes cluster with Octopus.
+
+*When you use Kustomize for one application configuration*
```

Comment:
@nikita-dergilev, this feels a bit out of context here. Can you expand on what you mean?

Reply:
@johnsimons, I meant to add another scenario — using Kustomize to template multiple apps. So the part here was to separate cases when you use Kustomize for one app from cases where you use it for many (see below).

It looks like I didn't do great job explaining it :-)

Reply:
cool